modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadDeferLaws.kt

```diff
@@ -31,24 +25,13 @@ object MonadDeferLaws {
     testStackSafety: Boolean = true
   ): List<Law> =
     BracketLaws.laws(SC, EQ, EQ_EITHER, EQERR) + listOf(
-      Law("MonadDefer bind: binding blocks") { SC.asyncBind(EQ) },
-      Law("MonadDefer bind: binding failure") { SC.asyncBindError(EQERR) },
-      Law("MonadDefer bind: unsafe binding") { SC.asyncBindUnsafe(EQ) },
-      Law("MonadDefer bind: unsafe binding failure") { SC.asyncBindUnsafeError(EQERR) },
-      Law("MonadDefer bind: binding in parallel") { SC.asyncParallelBind(EQ) },
-      Law("MonadDefer bind: binding cancellation before flatMap") { SC.asyncCancellationBefore(EQ) },
-      Law("MonadDefer bind: binding cancellation after flatMap") { SC.asyncCancellationAfter(EQ) },
-      Law("MonadDefer bind: bindingInContext cancellation before flatMap") { SC.inContextCancellationBefore(EQ) },
-      Law("MonadDefer bind: bindingInContext cancellation after flatMap") { SC.inContextCancellationAfter(EQ) },
-      Law("MonadDefer bind: bindingInContext throw equivalent to raiseError") { SC.inContextErrorThrow(EQERR) },
-      Law("MonadDefer bind: monad comprehensions binding in other threads equivalence") { SC.monadComprehensionsBindInContextEquivalent(EQ) },
```

Comment:
Where did these go?

Reply:
These were there for all the `bind` overloads that were removed from `MonadDefer`. They are gone.

Reply:
These functions were all invalid. They defined `Async` ops in `MonadDefer`.

The global cancelation is removed all together since it breaks RT.

Reply:
I.e. `bindIn(ctx) {..}` becomes `effect(ctx) {..}.bind()` etc